media/css/firefox/family/components/modules/_privacy.scss

```diff
@@ -48,7 +48,7 @@
             }
 
             .c-content {
-                grid-row: 2;
+                grid-row: 2 / -1;
```

Comment:
Not strictly related to this change, but there was a lot of extra space in the privacy section 2nd column, this makes the text in the 1st column span all subsequent rows, so it shouldn't create extra space between the browser and blurb
<img width="980" alt="spacing-fix" src="https://user-images.githubusercontent.com/19650432/194320156-47d90472-2241-4881-b185-b22bbf416bc0.png">